City-of-Helsinki/drupal-helfi-platform-config
# UHF-9514: TFA

## How to install
- `composer require drupal/helfi_platform_config:dev-UHF-9514`

## How to test
- Enable `helfi_tfa` module: `drush en helfi_tfa`
- Make sure all roles except authenticated and read_only roles are required to setup TFA
